assets/training/vision/components/image_classification/spec.yaml

```diff
@@ -188,7 +188,7 @@ outputs:
 
 code: ../src
 
-environment: azureml://registries/azureml/environments/automl-dnn-vision-gpu/versions/1
+environment: azureml://registries/azureml/environments/automl-dnn-vision-gpu/versions/3
```

Comment:
Nikolay released this env with version 2, that is still pointing to 1.53.0. Hence skipping that version and using the next version 3 here.